pna1.p4

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2022-present Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#ifndef __PNA1_P4__
+#define __PNA1_P4__
+
```

Comment:
Should we not edit pna.p4 instead of adding pna1.p4 ? We may  remove PreControl from pan.p4 and a new pna2.p4 file that expresses second architecture with pre-preparer and pre-control.